src/features/defi/providers/idle/components/IdleManager/Overview/IdleOverview.tsx

```diff
@@ -189,6 +210,21 @@ export const IdleOverview: React.FC<IdleOverviewProps> = ({
     ]
   }, [contractAddress, idleInvestor, opportunityData, hasClaimBalance, translate])
 
+  const renderTags = useMemo(() => {
+    return opportunityData?.tags?.map(tag => {
+      if (IdleTagDescriptions[tag as Tags]) {
+        const tagDetails = IdleTagDescriptions[tag as Tags]
```

Comment:
Once the `OpportunityMetadata` type is updated we shouldn't need to do this casting.